File: src/code-samples/langsmith/smithdb-migration/experiment-runs-query-basic-before.go
// :snippet-start: experiment-runs-query-basic-before-go
// :codegroup-tab: Before
package main

import (
	"context"

	"github.com/langchain-ai/langsmith-go"
)

// :remove-start:
var datasetID = "00000000-0000-0000-0000-000000000000"
var experimentID = "00000000-0000-0000-0000-000000000001"

func main() {
	if false {
// :remove-end:
ctx := context.Background()
client := langsmith.NewClient()

examplesWithRuns, err := client.Datasets.Runs.New(ctx, datasetID, langsmith.DatasetRunNewParams{
	SessionIDs: langsmith.F([]string{experimentID}),
	Limit:      langsmith.F(int64(20)),
	Preview:    langsmith.F(true),
})
// :remove-start:
if err != nil {
	panic(err.Error())
}
_ = examplesWithRuns
	}
}
// :remove-end:
// :snippet-end:

File: src/code-samples/langsmith/smithdb-migration/experiment-runs-query-basic-before.ts
// :snippet-start: experiment-runs-query-basic-before-js
// :codegroup-tab: Before
import { Client } from "langsmith";

// :remove-start:
const datasetId = "00000000-0000-0000-0000-000000000000";
const experimentId = "00000000-0000-0000-0000-000000000001";
// :remove-end:
const client = new Client();
// :remove-start:
if (false) {
// :remove-end:
const examplesWithRuns = await client.datasets.runs.query(datasetId, {
  session_ids: [experimentId],
  limit: 20,
  preview: true,
});
// :remove-start:
}
// :remove-end:
// :snippet-end:
